include/votca/xtp/qmpackage.h

```diff
@@ -118,6 +118,23 @@ class QMPackage {
   std::vector<std::string> GetLineAndSplit(std::ifstream& input_file,
                                            const std::string separators) const;
 
+  void ReorderMOsToXTP(Eigen::MatrixXd& v, const AOBasis& basis) const;
```

Comment:
I still think that for people new to the code a brief comment explaning what those methods do would be very useful :)